README.md

```diff
@@ -6,7 +6,7 @@
 
 The open-source cloud asset inventory powered by SQL.
 
-![BuildStatus](https://img.shields.io/github/workflow/status/cloudquery/cloudquery/test?style=flat-square)
+![BuildStatus](https://img.shields.io/github/workflow/status/cloudquery/cloudquery/Build?style=flat-square)
```

Comment:
Refers to this https://github.com/cloudquery/cloudquery/blob/main/.github/workflows/test_build.yml#L1